uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/controllers/sub-controllers/leave-request.controller.js

```diff
@@ -5,23 +5,48 @@ define([
 ], function (controllers) {
   controllers.controller('LeaveRequestCtrl', LeaveRequestCtrl);
 
-  LeaveRequestCtrl.$inject = ['$log', '$q', 'parentCtrl'];
+  LeaveRequestCtrl.$inject = ['$controller', '$log', '$q', 'parentCtrl'];
```

Comment:
instead of a generic `parentCtrl` let's have a more meaningful name, so we don't get confused of what parent we are talking about. Something like `detailsController`?

Reply:
Done.